GOVERNANCE.md

```diff
@@ -109,21 +117,54 @@ Before each TSC meeting, the meeting chair will share the agenda with members of
 the TSC. TSC members can also add items to the agenda at the beginning of each
 meeting. The meeting chair and the TSC cannot veto or remove items.
 
-The TSC may invite people to take part in a non-voting capacity.
-
-During the meeting, the TSC chair ensures that someone takes minutes. After the
-meeting, the TSC chair ensures that someone opens a pull request with the
-minutes.
-
-The TSC seeks to resolve as many issues as possible outside meetings using
-[the TSC issue tracker](https://github.com/nodejs/TSC/issues). The process in
-the issue tracker is:
+The TSC may invite people to take part in a non-voting capacity in either the
+public or private portions of the meeting.
+
+During the public portion of the meeting, the TSC chair ensures that someone
+takes minutes that include a summary of the discussion and any
+decisions made. After the meeting, the TSC chair ensures that someone opens a
+public pull request with the minutes from the public portion of the meeting.
+
+The public portion of the TSC meeting is expected to be recorded and made
+available for live streaming during the meeting or download by anyone after.
+This expectation is to be announced to all participants at the start of the
+each meeting before the recording is started. Continued participation in the
+public portion of the meeting after this announcement implies consent to the
+recording.
+
+For the private portion of the meeting, the TSC chair ensures that someone
+documents only the final disposition of any decisions made during the meeting
+and ensures that those are shared with all TSC members via a private discussion
+channel such as the TSC private email list. These summaries are not to be made
+public unless all TSC members present during the meeting agree.
```

Comment:
```suggestion
For the private portion of the meeting, the TSC chair ensures that someone
produces a summary of the discussions, gets it reviewed by the attendees
before the meeting ends, and shares it to all the TSC members once approved
by the attendees via a private discussion channel such as the TSC private
mailing list. The summary may be made public if there is consensus within the
TSC to make it public.
```

I think this is closer to what we have already been doing e.g. the chair posts a quick draft of the summary to the Zoom chat, either written by themselves or someone just verbally draft it in the meeting, and ask for objections/feedback on the spot before the meeting ends to post it afterwards.

Reply:
Re: this part of the suggestion: "The summary may be made public if there is consensus within the TSC to make it public." ...This is not adequate, I think. The agreement needs to be among all the people who were in attendance, not just TSC members.

Reply:
The only other part here is the "before the meeting ends" ... it is most common that we end up running out of time, participants drop early, etc. It generally hasn't been possible for us to get these summaries reviewed before the end of the call. I don't think we want to try to mandate that they are, just from a practicality perspective.

Reply:
Dropping `before the meeting ends` SGTM, wasn't intentional to make it limited to be on-spot though I just observed that that's what we've been doing. I guess the chair would just figure it out somehow.